f3.go

```diff
@@ -177,11 +177,21 @@ func (m *F3) Start(startCtx context.Context) (_err error) {
 	// Try to start immediately if we have a manifest available and don't have to wait to
 	// bootstrap. That way, we'll be fully started when we return from Start.
 	if initialDelay == 0 {
+		// FIXME: this failing will kill whole F3 and prevent it from starting
+		// dont do that
```

Comment:
How so?

Reply:
Ah, you mean if there's an error in the config or something? Yeah, given the dynamic manifeset, we probably want to log and continue in this case.

Reply:
Yeah, for example in butterfly we had bootstrap 200, finality 900.
But it also is the case with initial power table moving outside of the 2k epochs and becoming unavailable.

Reply:
Hm. As long as we warn _very loudly_. I guess we'll likely rip-out all the dynamic manifest logic for the final release anyways.